packages/shared/src/components/squads/SquadPageHeader.tsx

```diff
@@ -68,15 +135,29 @@ export function SquadPageHeader({
           {squad.description}
         </p>
       )}
-      <SquadMemberShortList
```

Comment:
We don't need this here anymore, should now always be part of the SquadHeaderBar.